### ISSUE TITLE
Add support for specifying amount in !joingroup command

### ISSUE THREAD

**Issue by pushkinb** [3urobeat/steam-comment-service-bot#249]
**Please describe the bug**
What happened?

Hello, i have a problems when i use command !joingroup, when i use the command with parametrs "!joingroups <link> 100" script send me "Joining group '103582791473802010' with all bot accounts..." but he did not take into account that I specified the number of accounts I needed, and began to join the group all at once

**Full error stack trace**
If you've got an error, please post the **entire** error stack trace here!
![image](https://github.com/3urobeat/steam-comment-service-bot/assets/160387754/686660c0-0b05-43f3-b8bb-a997bb0efd04)

**Additional context**
Please add any other information here, if you have some.

**Comment by 3urobeat** [comment]
Hi, thanks for opening an issue!  

The `!joingroup` command does not support specifying an amount (as of now).  
You can see this in the [commands documentation](https://github.com/3urobeat/steam-comment-service-bot/blob/master/docs/wiki/commands_doc.md).

It would probably be a good idea to use a standardized format for all commands to reduce confusion.  
I think I'll consider changing all relevant commands to use the same `!cmd amount destination` format in v2.16.0.

Changing tag to enhancement. I'll leave the issue open and update if I have news to share.

**Comment by pushkinb** [comment]
this problems i find also in command "addfriend":
![image](https://github.com/3urobeat/steam-comment-service-bot/assets/160387754/81a84e3a-ec9d-4ada-a858-019873b0a253)

**Comment by 3urobeat** [comment]
Yep, both commands use all accounts and do not take an amount argument.  
It is documented in the Commands documentation.

**Comment by pushkinb** [comment]
oh, can i make take amount argument for this commands please?

**Comment by 3urobeat** [comment]
The commands `!joingroup` & `!addfriend` now take a `amount` parameter before the `ID` parameter, just like all other request commands.   
You can also provide the keyword "max" or "all" as the `amount` parameter to use all available accounts, just like the default behavior before.  
If the bot replies with "Joining ... with 0 bot accounts..." then obviously no accounts were eligible to join the group (= are already member of the group).

I'll leave this issue open until Version 2.16.0 is out on the master branch which implements these new features.

**Comment by 3urobeat** [comment]
Version 2.16 is now out on the master branch, which resolves this issue!
Check out the release notes [here](https://github.com/3urobeat/steam-comment-service-bot/releases/tag/2.16.0)!

Your bot will automatically update in the next 12 hours or on restart (unless you have disabled it).
